packages/firebase_crashlytics/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.2.0-dev.1
+
+* **Breaking change**: Migration to the new Firebase Crashlytics SDK. Follow the [migration guide](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android) (steps 1 and 2) to update your app.
```

Comment:
Would it perhaps be better to point to the location of the docs for this plugin? I notice from the link that it points to Android so the thing that comes to mind as a reader is does this only affect Android and what about iOS

Reply:
Specific steps to migrate are now covered explicitly in the migration guide:

![image](https://user-images.githubusercontent.com/5347038/91998275-ed15d480-ed32-11ea-8557-b2fa85d5ec3e.png)